docs/content/docs/usage/routing.md

```diff
@@ -159,3 +159,55 @@ export default (
   </Route>
 )
 ```
+
+## Manipulate history
+
+To create basics links, you need to use the `Link` component from `react-router`
+
+```
+import {Link} from "react-router"
+
+const Nav = () => (
+  <div>
+    // ...
+    <Link to="/">
+      {"Homepage"}
+    </Link>
+    // ...
+  </div>
+)
+```
+
+But maybe you need to manipulate history outside of `Link` context.
+
+If you already use `react-router` you know [browserHistory](https://github.com/ReactTraining/react-router/blob/master/docs/guides/Histories.md).
+
+`Phenomic` automatically inject it into `react-router` router history prop.
```

Comment:
Also it's more "Phenomic injects its own browserHistory instance as a react-router props.